src/web/server/htmlTemplate.ts

```diff
@@ -103,6 +103,34 @@ export const htmlTemplate = ({
 
     const twitterMetaTags = generateMetaTags(twitterData, 'name');
 
+    // Duplicated prefetch and preconnect tags from DCP:
+    const staticPrefetchUrls = [
+        `https://assets.guim.co.uk/`,
+        `https://i.guim.co.uk`,
+        `https://api.nextgen.guardianapps.co.uk`,
+        `https://hits-secure.theguardian.com`,
+        `//j.ophan.co.uk`,
+        `//ophan.theguardian.com`,
+        `//phar.gu-web.net`,
+        `//www.google-analytics.com`,
+        '//sb.scorecardresearch.com',
+    ];
+
+    const staticPreconnectUrls = [
+        `https://assets.guim.co.uk/`,
+        `https://i.guim.co.uk`,
+        `//interactive.guim.co.uk`,
+        `//www.google-analytics.com`,
+    ];
+
+    const prefetchTags = staticPrefetchUrls.map(
+        src => `<link rel="dns-prefetch" href="${src}">`,
```

Comment:
would be great to link to an explanation of the benefits of `dns-prefetch` https://developer.mozilla.org/en-US/docs/Web/Performance/dns-prefetch

Reply:
👍